src/WebApiGateway/Models/FilterCriteria/FilterCriteriaRequestModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace WebApiGateway.Models.FilterCriteria
+{
+    public class FilterCriteriaRequestModel : OrderCriteriaRequestModel
+    {
+        public List<Guid>? UserIds { get; set; }
+        public bool IsEnabled { get; set; }
```

Comment:
IsEnabled should be nullable. Because this filter has 3 states. 1) on/off 2) IsEnabled = true 3) IsEnabled  = false